Automattic/wp-calypso
# Update Help Center Launchpad to use calypso-config import instead of window.configData and added try catch

#### Test/Review estimation:
Test: short
Review: short

#### Proposed Changes

* Replace `window.configData` usage (we shouldn't directly access this value) with calypso-config utility.
* Added try - catch around launchpad URL generation logic to prevent errors and default to `wordpress.com` environment if any exceptions are thrown.

#### Testing Instructions



* Checkout this branch
* Sandbox public API and a launchpad enabled site
* Navigate to Calypso apps/editing-toolkit and execute yarn dev --sync
* Navigate to the site-editor and click the question mark icon on the top right navigation bar to open the help center.
* Verify you are able to see the Launchpad entry point and that it navigates to the launchpad screen correctly.
* Test this out on another screen such as the /posts page.

<img width="439" alt="image" src="https://user-images.githubusercontent.com/10482592/215506244-f2c76a2f-3b7a-432e-bebd-11947a10fa6b.png">

#### Pre-merge Checklist





Related to #72276
